po/es.po

```diff
@@ -2647,55 +2642,56 @@ msgid ""
 "call functions such as `slice::get_unchecked` which does not do bounds "
 "checking."
 msgstr ""
-"El chequeo de límites no puede ser desactivado con el flag de compilador. "
-"Tampoco puede deshabilitarse directamente usando `unsafe`. Sin embargo, "
-"`unsafe` permite que llames a la función como `slice::get_unchecked` que no "
-"brinda chequeos de límites."
+"La comprobación de límites no se puede inhabilitar con una bandera de "
+"compilador. Tampoco se puede inhabilitar directamente con la palabra clave "
+"`unsafe`. Sin embargo, `unsafe` te permite llamar a funciones como `slice::"
+"get_unchecked`, que no realiza comprobaciones de límites."
 
 #: src/why-rust/modern.md:3
-#, fuzzy
 msgid "Rust is built with all the experience gained in the last decades."
-msgstr "Rust está hecho con toda la experiencia ganada en 40 años."
+msgstr ""
+"Rust se ha creado a partir de toda la experiencia obtenida en las últimas "
+"décadas."
 
 #: src/why-rust/modern.md:5
 msgid "Language Features"
 msgstr "Características del Lenguaje"
 
 #: src/why-rust/modern.md:7
 msgid "Enums and pattern matching."
-msgstr "Enums y pattern matching."
+msgstr "Enumeraciones (_Enums_) y coincidencia de patrones."
 
 #: src/why-rust/modern.md:8
 msgid "Generics."
-msgstr "Genericos."
+msgstr "Genéricos"
 
 #: src/why-rust/modern.md:9
 msgid "No overhead FFI."
-msgstr "No overhead FFI."
+msgstr "No hay sobrecarga de FFI"
```

Comment:
'sobrecarga' is 'overload', not overhead. No overhead FFI means no added performance penalty when calling FFI.